docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -50,13 +53,17 @@ testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
   * [--proxy \<host\>](#--proxy-host)
   * [--proxy-bypass \<rules\>](#--proxy-bypass-rules)
   * [--ssl \<options\>](#--ssl-options)
+  * [-L, --live](#-l---live)
   * [--dev](#--dev)
   * [--qr-code](#--qr-code)
   * [--sf, --stop-on-first-fail](#--sf---stop-on-first-fail)
-  * [--disable-test-syntax-validation](#--disable-test-syntax-validation)
   * [--color](#--color)
   * [--no-color](#--no-color)
 
+When you execute the `testcafe` command, TestCafe first reads settings from the `.testcaferc.json` [configuration file](configuration-file.md) if this file exists. Then it applies settings provided in the command line. Command line settings override values from the configuration file in case they differ. TestCafe prints information about every overridden property in the console.
```

Comment:
When you execute the `testcafe` command, TestCafe first reads settings from the `.testcaferc.json` [configuration file](configuration-file.md) if this file exists, and then applies the settings from the command line.